[Merge] 重构站点信息模块：初始化数据、优化接口、引入 DTO 传输

fanfan187 opened the pull request:
## PR 标题

[Merge] 重构站点信息模块：初始化数据、优化接口、引入 DTO 传输

---

## 变更内容

- 新增：
    - 新增 SiteDTO 类，用于解耦接口响应与数据库实体。
    - 在 Initialize 模块中预置站点信息数据。
- 修改：
    - 重构 PublicController 的接口，通过 SiteInfoService 聚合数据并返回统一响应。
- 优化：
    - 优化SiteInfoService。
- 删除：
    - 移除 SiteInfoService 中与 SiteDO 直接绑定的冗余逻辑。

fanfan187 committed:
feat(site): 新增获取站点基本信息接口

- 新增PublicController以输出系统基本信息
- 新增ResponseWrapperVO,SiteDO,SiteInfoService
- 对启动类，RedisConfig,TableMapper缓存部分进行修改

fanfan187 committed:
fix(site): 对获取站点基本信息接口进行修正

- 在 Initialize 中构建站点信息数据
- 完善 PublicController 获取站点信息逻辑
- 新增 SiteDTO 以传输数据
- 去除 SiteInfoService 中的冗余部分，返回SiteDTO站点信息

xiaolfeng commented on src/main/java/com/frontleaves/scheduling/ClassSchedulingApplication.java:
```diff
@@ -55,6 +56,7 @@
         com.xlf.utility.config.app.MybatisPlusConfiguration.class
 }, value = "com.frontleaves.scheduling")
 @EnableTransactionManagement
+@EnableCaching
```
项目不需要 EnableCaching 有关该注解撤销（一般情况下，不要编辑 Application 主类）

xiaolfeng commented on src/main/java/com/frontleaves/scheduling/controllers/PublicController.java:
```diff
@@ -0,0 +1,40 @@
+    private final SiteInfoService siteInfoService;
+
+    //获取站点信息接口
+    @GetMapping("/info")
```
编写 javadoc，javadoc 按照
/**
 * xxx
 * \<p>
 * 具体描述
 * 
 * @return 返回值
 * @param 形参（没有忽略）
 */

xiaolfeng commented on src/main/java/com/frontleaves/scheduling/mappers/TableMapper.java:
```diff
@@ -43,4 +46,5 @@
  */
 @Mapper
 public interface TableMapper extends BaseMapper<TableDO> {
+    List<SiteDO> getSiteInfo();
```
多余无用代码删除

xiaolfeng commented on src/main/java/com/frontleaves/scheduling/models/entity/SiteDO.java:
```diff
@@ -0,0 +1,29 @@
+package com.frontleaves.scheduling.models.entity;
```
无用DO 删除

xiaolfeng commented on src/main/java/com/frontleaves/scheduling/models/vo/ResponseWrapperVO.java:
```diff
@@ -0,0 +1,19 @@
+package com.frontleaves.scheduling.models.vo;
```
无用代码删除

xiaolfeng reviewed (changes requested):
有一些问题，修改后重新推送

fanfan187 committed:
fix(site): 对合并过程中已知错误进行更改

- 删除 ClassSchedulingApplication 中@EnableCaching注解
- 完善 PublicController 获取站点信息接口的javadoc注释
- 删除无用类 ResponseWrapperVO,SiteDO
- 删除 TableMapper 中无用代码

xiaolfeng committed:
refactor(standard): 重构站点信息获取逻辑

- 将 `SiteInfoService` 重命名为 `PublicLogic` 并实现 `PublicService` 接口
- 更新 `PublicController` 中的依赖注入，使用 `PublicLogic` 替代 `SiteInfoService`
- 移除 `RedisConfig` 中未使用的 `@EnableCaching` 注解
- 添加新的 `PublicService` 接口，定义 `getSiteInfo` 方法
